aipg/task_inference/task_inference.py

```
@@ -114,68 +121,61 @@ def transform(self, state: ProjectAgentState) -> ProjectAgentState:
+            last_exception: OutputParserException | None = None
+            for attempt in range(1, 4):
+                response = await self.llm.query(chat_prompt)
+                try:
+                    state.project = self.prompt_generator.parser(response)
+                    break
+                except OutputParserException as e:
+                    last_exception = e
+                    error_feedback = (
+                        f"Ошибка парсинга: {e}\n\n"
+                        "ВАЖНО: Ответь ТОЛЬКО чистым markdown без дополнительных объяснений или комментариев. "
+                        "Начни сразу с заголовка '# Микропроект для углубления темы: ...' "
```

Ruff (0.12.2)

String contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF001)